x/interchainstaking/keeper/validators.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // GetValidators returns validators by chainID.
 func (k Keeper) GetValidators(ctx sdk.Context, chainID string) []types.Validator {
-	validators := []types.Validator{}
```

Comment:
What is best practice here?

```
validators := []types.Validator{}
var validators []types.Validator
validators := make([]types.Validator, 0)
```

we should aim to be consistent, whatever the choice.

Reply:
i think the `var` declaration makes sense for 0 len slices you are building using `append()`.  And then only use `make()` when you are allocating something with actual `len` or `cap`